deps.ffmpeg/30-libvpx.ps1

```diff
@@ -40,7 +41,7 @@ function Configure {
         '../configure'
         ('--prefix="' + $($script:ConfigData.OutputPath -replace '([A-Fa-f]):','/$1' -replace '\\','/') + '"')
         ('--target=' + $($BuildTargets[$Target]))
-        '--enable-runtime-cpu-detect'
```

Comment:
Why is this removed?

Reply:
There had been issues with it when I started porting to ARM64 on Windows, but I cannot reproduce any compile issues with it anymore. Maybe issues will exhibit themselves at runtime (as e.g. Google themselves always disable runtime detection for their own ARM64 builds on Android), but I haven't been able to run into any yet.